ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -198,7 +201,49 @@ func (r *RayClusterReconciler) reconcileServices(instance *rayiov1alpha1.RayClus
 			return err
 		}
 
-		err = r.createHeadService(rayHeadSvc, instance)
+		err = r.createService(rayHeadSvc, instance)
+		// if the service cannot be created we return the error and requeue
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (r *RayClusterReconciler) reconcileDashBoardServices(instance *rayiov1alpha1.RayCluster) error {
```

Comment:
Would it be desirable to use some common code with the existing headService reconciler?
(It might not be, just asking.)

Reply:
We could let them share a method with some input parameter.

Reply:
Let's do that if it's not too awkward.